interface/models.py

```diff
@@ -30,6 +30,10 @@ class Assignment(models.Model):
     def full_code(self):
         return f'{self.course.code}-{self.code}'
 
+    @property
+    def submission_set(self):
```

Comment:
Wait, `submission_set` is already defined by Django, because of the `ForeignKey` from `Submission` to `Assignment`. You're duplicating Django's magic.

Reply:
I don't get it. Explain it in more depth please. Given an `Assignment` i need to get all of it's corresponding `Submissions`. How should it look like?

Reply:
https://docs.djangoproject.com/en/2.2/ref/models/relations/ <- Django creates a `submission_set` RelatedManager on any `Assignment` instance.